lib/pbench/server/api/resources/intake_base.py

```diff
@@ -153,7 +153,7 @@ def _stream(self, intake: Intake, request: Request) -> Access:
         """
         raise NotImplementedError()
 
-    def _cleanup(self, args: ApiParams, intake: Intake) -> list[str]:
+    def _cleanup(self, args: ApiParams, intake: Intake, notes: list[str]) -> list[str]:
```

Comment:
The return type hint needs correcting (removing).  (You caught it in the subclass.  🙂)